src/pimp_mixer_arm.S

```diff
@@ -20,15 +20,14 @@
 @irq-safe:   1121327
 @irq-unsafe: 1105083
 
-// #define IRQ_SAFE
+#define IRQ_SAFE /* on by default */
```

Comment:
I think it would be better to turn this into a define in the build-system than defining it in this source. And I think it would be good to make this a separate commit from fixing the logic.

Reply:
Right. But should this be added to `pimp_config.h`, or should the macros there also be configurable from the build system? (Or am I using it incorrectly? I'm just calling the Makefile, is there other stuff I should do before that?)

Reply:
Ah, `pimp_config.h` is probably the proper place for this. My bad.